regression/scripts/nonuniform_propeller_inflow/nonuniform_propeller_inflow.py

```diff
@@ -0,0 +1,143 @@
+# nonuniform_propeller_inflow_regression.py
+#
+# Created: Mar 2021, R. Erhard
+# Modified:
+
+import SUAVE
+from SUAVE.Core import Units
+from SUAVE.Methods.Propulsion import propeller_design 
+from SUAVE.Plots.Propeller_Plots import plot_propeller_performance
+
+import numpy as np
+import pylab as plt
+
+
+
+def main():
+    '''
+    This example shows a propeller operating in a nonuniform freestream flow.
+    A wing in front of the propeller produces a wake, which is accounted for in the propeller analysis.
+    
+    Associated branch: feature-nonuniform_freestream
+    '''
+    # setup basic propeller
+    Nr = 101
```

Comment:
If it can run with less and converge you should use less points. We're not looking for accuracy in a regression, speed is more important as long its consistent.